### DISCUSSION TITLE
Question: Add tag to custodian lambda function

### DISCUSSION THREAD

**Discussion by SteveL1** [cloud-custodian/cloud-custodian#7439] [AWS]
### Ask your question

Can anyone tell me if it possible for have custodian add a tag to resources that get created?

I can in the schema that policies can have a tags element, but it doesn't seem to add the tag to created lambda function

Thanks

**Comment by SteveL1** [comment]
Sorry, I was putting the tags section at the wrong level in the yaml

**Reply by swapHR** [reply]
Hey @SteveL1 did you resolved this tagging stuff for the resources being provisioned by custodian deployed policies? If yes can you share the resolution  here?

**Reply by SteveL1** [reply]
sorry, i didn't manage to get CC to tag the eventbridge resources, just the lambdas

**Reply by swapHR** [reply]
can you share how it did worked for lambda?

**Comment by SteveL1** [comment]
I've found that the tags get added to the lambda function , that's great. However, the cloudwatch eventbridge that initiates the lambda isn't tagged.  Is it possible to ensure that all custodian created resources are tagged?
